x/zkauth/ante/zkauth.go

```diff
@@ -152,7 +154,63 @@ func (zdf ZKAuthDeductFeeDecorator) AnteHandle(ctx sdk.Context, tx sdk.Tx, simul
 		return zdf.dfd.AnteHandle(ctx, tx, simulate, next)
 	}
 
-	// Case of zkauth msg, does nothing in this case
+	// Almost no different from the implementation of zdf.dfd.AnteHandle, uses ZKauthAddress for feePayer.
+	feeTx, ok := tx.(sdk.FeeTx)
+	if !ok {
+		return ctx, sdkerrors.Wrap(sdkerrors.ErrTxDecode, "Tx must be a FeeTx")
+	}
+
+	if addr := zdf.ak.GetModuleAddress(authtypes.FeeCollectorName); addr.Empty() {
+		return ctx, fmt.Errorf("fee collector module account (%s) has not been set", authtypes.FeeCollectorName)
+	}
+
+	fee := feeTx.GetFee()
+	signature := zkMsgs[0].GetZkAuthSignature()
```

Comment:
I'm not sure, so I have a question, is it not necessary to run a loop with zkMsgs?Is zkMsgs[0] enough?

Reply:
The honest answer is that the specifications have not been determined at the moment.
I would like to note in the comments that it is necessary to consider how to respond when multiple zkmsg are included in one tx.

Reply:
And we need to check the case that the tx messagea are mixed both zkauth and other normal messages.